coverage_point_calculator/src/hexes.rs

```diff
@@ -0,0 +1,90 @@
+use coverage_map::RankedCoverage;
+use hex_assignments::assignment::HexAssignments;
+use rust_decimal::Decimal;
+use rust_decimal_macros::dec;
+
+use crate::{RadioType, Result};
+
+#[derive(Debug, Clone)]
+pub struct CoveredHexes {
+    pub hexes: Vec<CoveredHex>,
+}
+
+#[derive(Debug, Clone)]
+pub struct CoveredHex {
+    pub hex: hextree::Cell,
+    // --
```

Comment:
leave regular comments out of structs, unless they are doc comments, _which is more than encouraged_ (but they have to be actual descriptions of the field)

Reply:
Comments were added, thanks for the push.